helmfile/charts/prometheus-alerts/templates/alerts/kubernetes-apps.yaml

```
@@ -53,7 +53,7 @@ spec:
       annotations:
         description: Container {{`{{`}} $labels.container {{`}}`}} in pod {{`{{`}} $labels.namespace {{`}}`}}/{{`{{`}} $labels.pod {{`}}`}} from {{`{{`}} $labels.cluster {{`}}`}} has been OOMKilled {{`{{`}} $value {{`}}`}} times in the last 10 minutes.
         summary: Kubernetes container OOMKilled.
-      expr: (kube_pod_container_status_restarts_total - kube_pod_container_status_restarts_total offset 10m >= 1) and ignoring (reason) min_over_time(kube_pod_container_status_last_terminated_reason{reason="OOMKilled"}[10m]) == 1
+      expr: (kube_pod_container_status_restarts_total - kube_pod_container_status_restarts_total offset 10m >= 2) and ignoring (reason) min_over_time(kube_pod_container_status_last_terminated_reason{reason="OOMKilled"}[10m]) == 1
```

I think we should increase the time limit so we don't miss pods that are going oom often but not more than once every 10 minutes. I suggest increasing the time window to 30 min, but I'm open for suggestions on other lengths.
```suggestion
      expr: (kube_pod_container_status_restarts_total - kube_pod_container_status_restarts_total offset 30m >= 2) and ignoring (reason) min_over_time(kube_pod_container_status_last_terminated_reason{reason="OOMKilled"}[30m]) == 1
```

---

Thanks @viktor-f I have increased the time range accordingly